extensions/classifiers/LDAStringClassifier/LDAStringClassifier.py

```diff
@@ -515,6 +516,7 @@ def from_dict(self, model):
         Args:
             model: A dict representing a StringClassifier.
         """
+        model = self._unpickle_json_serialization(model)
```

Comment:
I did not review this part of the code, and, in fact I wanted to talk to you about it. As things stand, we might end up not using the LDAStringClassifier. So, does it makes sense to make any more changes to this part of the code (other than what is already there).